.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```

Comment:
Btw, have you tried without that sleep? Maven needs to do a lot of things before X server will be needed.